CHANGELOG.md

```diff
@@ -0,0 +1,77 @@
+# Drafter Changelog
+
+## Master
+
+### Bug Fixes
+
+* Parameters in Refract output will no longer contain a `content` value
+  when there is no sample value.  
+  [#286](https://github.com/apiaryio/drafter/issues/286)
+
```

Comment:
There we're no user facing memory leaks, they we're purely in the tests themselves so they don't warrant going in the CHANGELOG.

#138 and #243 we're bug in Snowcrash and not Drafter, so they should go into Snowcrash changelog.